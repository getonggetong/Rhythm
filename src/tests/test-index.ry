init() {
	
	a = [A4,B4,C4];
	print(a[1]);
	a = [A4,B4,[C4,D4,E4]];
	print(a[2][1]);

}