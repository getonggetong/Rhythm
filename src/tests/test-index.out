B4
D4